### ISSUE TITLE
Figure out how EV3 sensors work in leJOS

### ISSUE THREAD

**Issue by daanjels** [arqetype/Lejos#14]
Read input from the touch sensor

Read input from the light sensor and color sensor

**Comment by daanjels** [comment]
Touch sensor is quite easy to do. I need to think of a nice application to make it bit more interesting. Maybe something with sounds. I haven't done that yet. I'll put it in the project as a separate task.

**Comment by daanjels** [comment]
Playing with the lightsensor was nice. Getting the values is a bit strange at first because you need to create a sample that contects to the sensor. Once that is done the sample updates continuously. Making it easier to use after the initial setup.
I created a class that reads the color input. I first tried to display the sensor input as RGB values, by multiplying the values (from 0 to 1) by 256. But then I discovered that the values are always pretty low. So by trial and error I found the values to read the colour of lego pieces.
Later on I will compare it with the predefined colors in leJOS.
